<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IHM Bus CAN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #FFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>901</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item row="2" column="0">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="MotorBox" native="true">
       <widget class="QWidget" name="DistanceBrightnessBox" native="true">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>-10</y>
          <width>441</width>
          <height>591</height>
         </rect>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Noto Sans</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>true</italic>
           <bold>false</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 italic 16pt &quot;Noto Sans&quot;; 
text-decoration: underline;
color: rgb(128, 128, 168);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="OnOffButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>260</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #74D4E2;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="autoToggleButton">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>260</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #FD9501;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>mode auto</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="MotorSpeed">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>200</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Vitesse perso</string>
         </property>
        </widget>
        <widget class="QWidget" name="SensorValues" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>-20</y>
           <width>431</width>
           <height>201</height>
          </rect>
         </property>
         <widget class="QGroupBox" name="windSpeedvalue">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>341</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>Vitesse vent (Km/h)</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <widget class="QLabel" name="windSpeedLabel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>63</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>30</y>
            <width>221</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Contrôle du Moteur DYN_ANEMO</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="setSpeed">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>200</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(119, 118, 123);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="threeDbox" native="true">
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>300</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <widget class="QRadioButton" name="BrightnessRadio">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>260</y>
          <width>112</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Luminosité</string>
        </property>
       </widget>
       <widget class="QLabel" name="uniteLabel">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>220</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="DistanceBrightnessValue">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>-10</y>
           <width>311</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="title" stdset="0">
          <string>Mode Capteur</string>
         </property>
        </widget>
       </widget>
       <widget class="QRadioButton" name="DistanceRadio">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>260</y>
          <width>112</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Distance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>VL6180X_PRESS_HUM_TEMP</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="HumidityValue">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>60</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Humidité (%)</string>
        </property>
        <widget class="QLabel" name="HumidityLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="TemperatureValue">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Température (°C)</string>
        </property>
        <widget class="QLabel" name="temperaturelabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="PressureValue">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Pression (hPa)</string>
        </property>
        <widget class="QLabel" name="PressureLabel">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>63</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="phiValue">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="phiLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ThetaValue">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="ThetaLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PsiValue">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>600</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="PsiLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>630</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Phi φ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Theta θ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>630</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Psi Ψ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MPU9250 :accéléromètre</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
